ch03.asciidoc

```diff
@@ -289,17 +289,21 @@ While this method could be fully executed faster than the good and the bad way t
 === Invoices
 
 Every payment on the Lightning Network starts with a person who wants to receive bitcoins issuing an invoice.
+An invoice is nothing more than a very simple payment instruction containing information such as a recipient and an amount.
```

Comment:
and the payment hash. actually the payment hash ist the most important pieace of information (:

Reply:
I wanted to keep it simple, but the hash is so crucial that it should be included. Good point.
see commit # 2 97b20db5e032f8fc56c243b2ee2c4818c8e1ec49